website/views/options/profile.jade

```diff
@@ -175,65 +175,64 @@ mixin customizeProfile(mobile)
 script(id='partials/options.profile.avatar.html', type='text/ng-template')
   +customizeProfile()
 
-mixin profileStats
+script(id='partials/options.profile.stats.html', type='text/ng-template')
   .container-fluid
-    div(class=mobile?'padding':'row')
-      // FIXME, get this working on mobile
+    div(class='row')
```

Comment:
This is not a big deal, so I'm just going to go ahead and merge this as is, but `.row` is equivalent to `div(class="row")` (truthfully, since the client code is being reworked soon, I should have just let the `mobile` logic slide entirely 😓  )